.flake8

```diff
@@ -7,3 +7,9 @@ exclude =
   node_modules
 per-file-ignores =
   *.py: A003
+  # Ignore E800 until https://github.com/myint/eradicate/pull/19 is merged
+  ./src/_stories/contract.py: E800
+  ./tests/*: S, T, E800
+accept-encodings = utf-8
+max-cognitive-complexity = 30
+max-expression-complexity = 10
```

Comment:
It's a nice plugin to have!

But there is no value in it until we allow it to do its job. If we set the threshold that big, how it will improve our code?

See `xenon` related comments.